src/client/util/parse.h

```diff
@@ -40,12 +40,10 @@
 
 namespace kovri {
 namespace client {
-
-/// @brief Parses CSV string
+/// @brief Parses an ACL list
 /// @param record String record of CSV
-/// @return Vector of parsed values
-const std::vector<std::string> ParseCSV(
-    const std::string& record);
+/// @return Returns a std::set of the b32 of each value from the list
+const std::set<kovri::core::IdentHash> ParseACLList(const std::string list);
```

Comment:
`ParseACLList` can simply be named `ParseACL` (access control list list is redundant).